packages/design/src/FormManager/FormEdit/Preview.tsx

```diff
@@ -104,31 +104,35 @@ const createPatternPreviewComponent = (
       ? 'form-group-row field-selected'
       : 'form-group-row';
 
+    const editButton = pattern.type === 'input' ? (
```

Comment:
Only show the edit button if it's an input, skip for purely presentational elements.

Reply:
I updated this hide when there is no edit component for the pattern defined.